iset.mm

```diff
@@ -62258,6 +62290,30 @@ the first case of his notation (simple exponentiation) and subscript it
       FGVOXACFVEVOXBBFABCVFVGVHVIBFVJNVPAFGZVRFGVOVNWIXCWJABFCVKVLAFVJNVM $.
   $}
 
+  $( Any set dominates the empty set.  (Contributed by NM, 26-Oct-2003.)
+     (Revised by Mario Carneiro, 26-Apr-2015.) $)
+  0domg $p |- ( A e. V -> (/) ~<_ A ) $=
+    ( wcel c0 wss cdom wbr 0ss ssdomg mpi ) ABCDAEDAFGAHDABIJ $.
+
+  ${
+    $d A x $.
+    $( A set dominated by the empty set is empty.  (Contributed by NM,
```

Comment:
A class is dominated by the empty set if and only if it is empty.

Reply:
Norm has a way of writing the comments to call out the part he thinks is most interesting (especially, as here, when the precise statement of the theorem is also quite easy to read). I'm not saying we should never revise his comments but I'm not sure this is a case where the revised version is obviously better (at least in my mind).

Reply:
Fine for this one, but I really think the comment should describe the formal statement and the formal statement should formalize the comment. The "most interesting part" is not so objective. There is a discrepancy between being so cautious as to formally proving $\phi \to \phi$ on one side, and being lax with comments on the other side. The problem is not for any one statement; the problem is that these approximations might add up, and at the end of a chain of statements, we don't know what the formal version is proving (without a lot of backtracking work).